### PR TITLE
[#3208] Package core components

### LINKED ISSUE #3208: Package all the components in individual helm charts

### PR DESCRIPTION
**Pull request by ljupcovangelski**
One main helm chart with the main components:
- airy-controller
- api-admin
- api-communication
- api-websocket
- frontend-inbox
- frontend-control-center

The main components cannot be installed/upgrade/uninstalled as a separate helm charts.

The rest of the components are packaged and pushed separately:
- api-contacts
- integration-source-api
- integration-webhook
- media-resolver
- sources-chatplugin
- sources-facebook
- sources-google
- sources-twilio
- sources-viber
- sources-whatsapp

Also the helm charts have been moved to the source directory of each component individually.

Resolves #3208.

### CODE REVIEW

File: backend/sources/api/helm/BUILD [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+)
+
+helm_template_test(
+    name = "template",
+    chart = ":package",
+)
+
+helm_push(
+    chart = ":package",
+)
```
Comment (chrismatix) [inline backend/sources/api/helm/BUILD:28]: nitpick: These lines seem to be the same everywhere. If we add a `helm_package` macro in the tools directory we can avoid rewriting them everywhere.

**Review by juan-sebastian — DISMISSED** [review]
Great work, besides @chrismatix comment. It looks good to me

File: backend/sources/api/helm/BUILD (hunk 1 above)
Comment (ljupcovangelski) [reply backend/sources/api/helm/BUILD:28]: Got it :+1:. Will do it that way.

**Review by AitorAlgorta — APPROVED** [review]
🚀

**Pull request merged**